### ISSUE TITLE
Early indicators for tracking training 

### ISSUE THREAD

**Issue by ShantanuNathan** [AlexeyAB/darknet#3295]
Hi @AlexeyAB ,

I've been using your repo for quite some time now and have been consistently getting 90% + mAP on my validation set. However with large my dataset, network size and other configurations (batch size = 64, subdivisions = 64) etc I am forced to wait for nearly 20-25 hours of training to get a model of the above accuracy. Is there any way or are there any indicators that could tell me whether the training is going well or not very early on itself?
I am specifically asking this because in my latest training session, even though the loss was converging well enough as observed in the previous sessions, the validation set accuracy was below par. 
I am training for 45 classes, nearly 1500 images with almost each of the classes occurring more than once in each of the images. The dataset that I had mentioned has not considerably varied across all these runs, not more than 5%.

**Comment by AlexeyAB** [comment]
@ShantanuNathan Hi,

Just train with flag ` -map` and look at mAP value (red-line), if mAP increases then training goes well: https://github.com/AlexeyAB/darknet#when-should-i-stop-training

**Comment by ShantanuNathan** [comment]
Thanks for your help
